_posts/2025-02-06-highlights-2024.md

```diff
@@ -0,0 +1,247 @@
+In addition, IntelliJ automatically recognizes it when importing existing Scala CLI projects.
+Other minor enhancements are planned for future releases.
+
+Throughout 2024, the Scala 3 support improved steadily with each release.
+Although there is still work to do, the latest IntelliJ Scala Plugin can now import most Scala 3 projects effectively.
+Furthermore, Scala 3.3.x LTS is now selected as the new default version when creating new projects.
+
+Other major updates from 2024 include:
+- [X-Ray Mode](https://blog.jetbrains.com/scala/2023/12/21/the-x-ray-mode/), enabling inlay hints to reveal inferred types, parameter names, and implicit arguments
+- Experimental support for transparent inline methods ([2024.3](https://blog.jetbrains.com/scala/2024/11/13/intellij-scala-plugin-2024-3-is-out/#transparent-inline-methods-experimental))
+- Support for named tuples ([2024.3](https://blog.jetbrains.com/scala/2024/11/13/intellij-scala-plugin-2024-3-is-out/#named-tuples))
+
+As a further read, check out the [The IntelliJ Scala Plugin in 2024](https://blog.jetbrains.com/scala/2024/12/20/the-intellij-scala-plugin-in-2024/) post on JetBrains' blog.
+
+### Best-effort compilation in Metals
+
+Our main focus with Metals has been stability, to provide a smoother experience for Scala developers.
+One of those efforts has been implementing best-effort compilation for Scala 3.
+
+Best-effort compilation allows Metals to provide the most up-to-date information, for things such as autocompletion even in broken code that fails to compile.
+
+Since this is a big change, we are still making sure everything works correctly.
+The feature remains opt-in for now.
+To try it out you can start Metals with `-Dmetals.enable-best-effort=true`, or if using VS Code put that into `metals.serverProperties`.
+
+A more limited version is implemented for Scala 2. It is based on outline compilation, which only compiles signatures.
+The main limitation is that it only works inside the broken module, not in its dependent modules.
+
+Other major improvements in Metals this year include:
+
+- Bazel support ([v1.2.1](https://scalameta.org/metals/blog/2024/02/07/bismuth/#bazel-support))
+- Inlay hints ([v1.3.0](https://scalameta.org/metals/blog/2024/04/15/thalium/#inlay-hints-support))
+- Go to implementations for dependency sources ([v1.3.0](https://scalameta.org/metals/blog/2024/04/15/thalium/#go-to-implementations-for-dependency-sources))
+- Debugging for all build servers ([v1.3.3](https://scalameta.org/metals/blog/2024/07/12/thallium/#allow-debugging-for-all-build-servers))
+- Release of Bloop 2 ([v1.4.0](https://scalameta.org/metals/blog/2024/10/24/palladium/#bloop-2))
+
+## Governance
+
+One of the Scala organization’s major efforts this year was to establish, formalize, and document our own operations, as well as the policies governing releases of the Scala language, as seen on the pages linked below.
```

Comment:
As discussed, I don't think `the Scala organization` should be referred to explicitly, as it gives the impression that there's a specific entity behind the efforts.

Perhaps this could be:

```
One of the major efforts this year was to establish ...
```